<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartEyeClass</class>
 <widget class="QMainWindow" name="SmartEyeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartEye V1.0.6</string>
  </property>
  <property name="windowIcon">
   <iconset resource="smarteye.qrc">
    <normaloff>:/SmartEye/Resource/exe.png</normaloff>:/SmartEye/Resource/exe.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Colormap</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="0">
          <widget class="QCheckBox" name="colormapPointCheckBox">
           <property name="text">
            <string>Clolormap Point</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>MaxDepth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="maxdepthlineEdit">
             <property name="text">
              <string>25000</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>MinDepth</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="mindepthlineEdit">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>PointCloud Convert</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="3" column="1">
              <widget class="QLineEdit" name="CYlineEdit">
               <property name="text">
                <string>133.6313985</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="k2lineEdit">
               <property name="text">
                <string>0.102422447</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="FXlineEdit">
               <property name="text">
                <string>191.8097556</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>CX</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="pointFilterEdit">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="k1lineEdit">
               <property name="text">
                <string>-0.293527263</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>k2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="CXlineEdit">
               <property name="text">
                <string>162.3485761</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Density</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>FY</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>k1</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>FX</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>CY</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="FYlineEdit">
               <property name="text">
                <string>190.5375098</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="pointSizeSlider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pclBtn">
             <property name="text">
              <string>PointCloud Convert</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>IP Address</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="1">
          <widget class="QLabel" name="statelabel">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(231, 231, 231);text-align: center;</string>
           </property>
           <property name="text">
            <string>NO</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>IP：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="IplineEdit">
             <property name="text">
              <string>192.168.7.94</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="PortlineEdit">
             <property name="text">
              <string>50660</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="connectButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Depth Information</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Dist(mm):</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QSpinBox" name="offsetSpinBox">
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>30000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="FramelineEdit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>ImgFrame</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="xlineEdit"/>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="disLineEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Offset</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Depth:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="ylineEdit"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="depthlineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="Savebutton">
             <property name="text">
              <string>Save PNG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="savestatelabel">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(231, 231, 231);</string>
             </property>
             <property name="text">
              <string>Saved</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="DepthImgMultiSave">
           <property name="text">
            <string>multi frame</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="savePcdButton">
             <property name="text">
              <string>Save PCD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="savePCDStateLable">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(231, 231, 231);</string>
             </property>
             <property name="text">
              <string>Saved</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="PclImgMultiSave">
           <property name="text">
            <string>multi frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="HFlip">
           <property name="text">
            <string>Horizontal Flip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="VFlip">
           <property name="text">
            <string>Vertical Flip</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>HFlip</zorder>
        <zorder>VFlip</zorder>
        <zorder></zorder>
        <zorder></zorder>
        <zorder>DepthImgMultiSave</zorder>
        <zorder>PclImgMultiSave</zorder>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Set Parametre</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="1">
          <widget class="QLineEdit" name="IntegrationtimeHDRlineEdit">
           <property name="text">
            <string>2500</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBoxHDR">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>HDR</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>MinAmp</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="IntegrationtimelineEdit">
           <property name="text">
            <string>500</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBoxSetABS">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;环境光校准&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>SetABS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="setAmplineEdit">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;小于此强度即设为0，不改变缩放下限&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="checkBoxAmp">
           <property name="text">
            <string>强度图</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBoxDRNU">
           <property name="text">
            <string>DRNU</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>TOF(us)1rd</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>TOF(us)2rd</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="setMaxAmplineEdit">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;所有强度不超过此限定值，即缩放上限&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>500</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>MaxAmp</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBoxCalibration">
           <property name="text">
            <string>畸变校正</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="pointCloudDock">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>400</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Point Cloud</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QVTKWidget" name="screen"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="imageDock">
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Depth Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Img_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000000</width>
         <height>10000000</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="locale">
        <locale language="Chinese" country="China"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ampDock">
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Amplitude Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QLabel" name="amp_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100000</width>
         <height>100000</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>pointCloudDock</zorder>
  <zorder>imageDock</zorder>
  <zorder>ampDock</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="smarteye.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>TCPSocketSlot()</slot>
  <slot>connectStateSlot()</slot>
  <slot>showImageSlot()</slot>
 </slots>
</ui>
